homeassistant/components/vallox/sensor.py

```diff
@@ -140,20 +150,40 @@ class ValloxSensorEntityDescription(SensorEntityDescription):
         icon="mdi:fan",
         state_class=SensorStateClass.MEASUREMENT,
         native_unit_of_measurement=PERCENTAGE,
-        sensor_type=ValloxFanSpeedSensor,
+        entity_type=ValloxFanSpeedSensor,
+    ),
+    ValloxSensorEntityDescription(
+        key="extract_fan_speed",
+        name="Extract fan speed",
+        metric_key="A_CYC_EXTR_FAN_SPEED",
+        icon="mdi:fan",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=FREQUENCY_HERTZ,
+        entity_type=ValloxFanSpeedSensor,
+        entity_registry_enabled_default=False,
```

Comment:
Why is this disabled by default?

Reply:
It shows Fan RPMs, I calculate amount of air pushed through ventilation. Do not think this is an interesting metric for generic audience.

Reply:
So, should it be marked as diagnostic instead?

Reply:
It is high-frequency always changing metric. I'd keep it away from DB.

Reply:
It always drifts a bit. Like: 1130 rpm, then 1128, then 1126, then 1150 and so on.